tracer/build/_build/BuildVariables.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using Nuke.Common;
+using Nuke.Common.IO;
+
+public partial class Build
```

Comment:
@andrewlock Added private methods in separate partial class, please review.
Hope this not an overkill 😄